string pwm_m1
string pwm_m2
string pwm_m3
string pwm_m4
string direccion
string comando